### ISSUE TITLE
Trident mount fail on OCP 4.4.6 trying to access ontap-nas

### ISSUE THREAD

**Issue by fckbo** [NetApp/trident#471]
**Description**
After installing trident, creating a back-end, a storage class & pvc/pv. Deploying a simple pod accessing the pv failed with the following error
```
  Warning  FailedMount             10s (x5 over 18s)  kubelet, worker3.mycluster.example.com  MountVolume.SetUp failed for volume "pvc-82dea51b-723b-49d6-b4e6-4331e7f76052" : kubernetes.io/csi: mounter.SetupAt failed: rpc error: code = Internal desc = error mounting NFS volume 10.85.86.29:/fbcluster_pvc_82dea51b_723b_49d6_b4e6_4331e7f76052 on mountpoint /var/lib/kubelet/pods/93e13576-5ba0-4283-af61-eb9af5f7682b/volumes/kubernetes.io~csi/pvc-82dea51b-723b-49d6-b4e6-4331e7f76052/mount: exit status 32
```

**Environment**

- Trident version: [20.07]
- Trident installation flags used: [./tridentctl install -n trident-test]
- Container runtime: [crio version 1.17.4-12.dev.rhaos4.4.git2be4d9c.el8]
- Kubernetes version: v1.17.1+f63db30
- Kubernetes orchestrator: [OCP 4.4.6]
- Kubernetes enabled feature gates: [e.g. CSINodeInfo]
- OS: [Red Hat Enterprise Linux CoreOS-44.81.202005250830-0]
- NetApp backend types: [ONTAP 9.7]
- Other:

**To Reproduce**
```
 ./tridentctl install -n trident-test
INFO Starting Trident installation.                namespace=trident-test
INFO Created namespace.                            namespace=trident-test
INFO Created service account.                     
INFO Created cluster role.                        
INFO Created cluster role binding.                
INFO Created Trident's security context constraint.  scc=trident user=trident-csi
INFO Created Trident pod security policy.         
INFO Added finalizers to custom resource definitions. 
INFO Created Trident service.                     
INFO Created Trident secret.                      
INFO Created Trident deployment.                  
INFO Created Trident daemonset.                   
INFO Waiting for Trident pod to start.            
INFO Trident pod started.                          namespace=trident-test pod=trident-csi-84cc6bc859-tbbs6
INFO Waiting for Trident REST interface.          
INFO Trident REST interface is up.                 version=20.07.1
INFO Trident installation succeeded.   
```
```
oc get pods -n trident-test
NAME                           READY   STATUS    RESTARTS   AGE
trident-csi-5k4hz              2/2     Running   0          17s
trident-csi-84cc6bc859-tbbs6   6/6     Running   0          17s
trident-csi-bljhc              2/2     Running   0          17s
trident-csi-hjxjc              2/2     Running   0          17s
trident-csi-lzmrr              2/2     Running   0          17s
trident-csi-s8hzz              2/2     Running   0          17s
trident-csi-xxrlz              2/2     Running   0          17s
```
```
./tridentctl -n trident-test version
+----------------+----------------+
| SERVER VERSION | CLIENT VERSION |
+----------------+----------------+
| 20.07.1        | 20.07.1        |
+----------------+----------------+
```
```
cat ../trident-test/fb-backend.json
{
    "version": 1,
    "storageDriverName": "ontap-nas",
    "backendName": "test-backend",
    "aggregate":"clu_ontap_dev_01_VM_DISK_2",
    "storagePrefix":"fbcluster_",
    "managementLIF": "10.85.86.28",
    "dataLIF": "10.85.86.29",
    "svm": "svm-dev-2",
    "username": "fbtrident",
    "password": "<mypassword>",
    "limitVolumeSize": "100Gi",
    "nfsMountOptions": "nfsvers=3"
}
```
```
./tridentctl -n trident-test create backend -f ../trident-test/fb-backend.json
+--------------+----------------+--------------------------------------+--------+---------+
|     NAME     | STORAGE DRIVER |                 UUID                 | STATE  | VOLUMES |
+--------------+----------------+--------------------------------------+--------+---------+
| test-backend | ontap-nas      | fca9322a-19ae-4656-8a16-2f6ed68de325 | online |       0 |
+--------------+----------------+--------------------------------------+--------+---------+
```
```
oc create -f ../trident-test/storage-class-basic.yaml
storageclass.storage.k8s.io/basic created
```
```
oc create -f ../trident-test/pvc-basic.yaml
persistentvolumeclaim/basic created
```
```
oc create -f ../trident-test/task-pv-pod.yaml
pod/task-pv-pod created
```

**Expected behavior**
pod should be running after mounting the pv

**Unexpected Behaviour**
```
oc get pods
NAME          READY   STATUS              RESTARTS   AGE
fb-pod        1/1     Running             0          4h22m
task-pv-pod   0/1     ContainerCreating   0          4s
```
```
oc describe pod task-pv-pod
Name:         task-pv-pod
Namespace:    default
Priority:     0
Node:         worker3.mycluster.example.com/192.168.10.32
Start Time:   Wed, 21 Oct 2020 11:55:21 +0200
Labels:       <none>
Annotations:  <none>
Status:       Pending
IP:           
IPs:          <none>
Containers:
  task-pv-container:
    Container ID:   
    Image:          nginx
    Image ID:       
    Port:           80/TCP
    Host Port:      0/TCP
    State:          Waiting
      Reason:       ContainerCreating
    Ready:          False
    Restart Count:  0
    Environment:    <none>
    Mounts:
      /usr/share/nginx/html from task-pv-storage (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from default-token-skvlr (ro)
Conditions:
  Type              Status
  Initialized       True 
  Ready             False 
  ContainersReady   False 
  PodScheduled      True 
Volumes:
  task-pv-storage:
    Type:       PersistentVolumeClaim (a reference to a PersistentVolumeClaim in the same namespace)
    ClaimName:  basic
    ReadOnly:   false
  default-token-skvlr:
    Type:        Secret (a volume populated by a Secret)
    SecretName:  default-token-skvlr
    Optional:    false
QoS Class:       BestEffort
Node-Selectors:  <none>
Tolerations:     node.kubernetes.io/not-ready:NoExecute for 300s
                 node.kubernetes.io/unreachable:NoExecute for 300s
Events:
  Type     Reason                  Age               From                                    Message
  ----     ------                  ----              ----                                    -------
  Normal   Scheduled               18s               default-scheduler                       Successfully assigned default/task-pv-pod to worker3.mycluster.example.com
  Normal   SuccessfulAttachVolume  16s               attachdetach-controller                 AttachVolume.Attach succeeded for volume "pvc-82dea51b-723b-49d6-b4e6-4331e7f76052"
  Warning  FailedMount             8s (x5 over 16s)  kubelet, worker3.mycluster.example.com  MountVolume.SetUp failed for volume "pvc-82dea51b-723b-49d6-b4e6-4331e7f76052" : kubernetes.io/csi: mounter.SetupAt failed: rpc error: code = Internal desc = error mounting NFS volume 10.85.86.29:/fbcluster_pvc_82dea51b_723b_49d6_b4e6_4331e7f76052 on mountpoint /var/lib/kubelet/pods/93e13576-5ba0-4283-af61-eb9af5f7682b/volumes/kubernetes.io~csi/pvc-82dea51b-723b-49d6-b4e6-4331e7f76052/mount: exit status 32
```
**Additional context**
```
./tridentctl -n trident-test logs
trident-controller log:
time="2020-10-21T09:43:36Z" level=info msg="Running Trident storage orchestrator." binary=/usr/local/bin/trident_orchestrator build_time="Tue Sep 22 20:20:23 UTC 2020" version=20.07.1
time="2020-10-21T09:43:36Z" level=info msg="Initializing metrics frontend." address=":8001"
time="2020-10-21T09:43:36Z" level=info msg="Added frontend." name=metrics
time="2020-10-21T09:43:36Z" level=info msg="Initializing HTTP REST frontend." address="127.0.0.1:8000"
time="2020-10-21T09:43:36Z" level=info msg="Added frontend." name="HTTP REST"
time="2020-10-21T09:43:36Z" level=info msg="Initializing HTTPS REST frontend." address=":8443"
time="2020-10-21T09:43:36Z" level=info msg="Added frontend." name="HTTPS REST"
time="2020-10-21T09:43:36Z" level=info msg="Initializing K8S helper frontend." namespace=trident-test
time="2020-10-21T09:43:36Z" level=info msg="K8S helper determined the container orchestrator version." gitVersion=v1.17.1+f63db30 version=1.17+
time="2020-10-21T09:43:36Z" level=info msg="Added frontend." name=k8s_csi_helper
time="2020-10-21T09:43:36Z" level=info msg="Initializing CSI frontend." name=worker2.mycluster.example.com version=20.07.1
time="2020-10-21T09:43:36Z" level=info msg="Enabling controller service capability." capability=CREATE_DELETE_VOLUME
time="2020-10-21T09:43:36Z" level=info msg="Enabling controller service capability." capability=PUBLISH_UNPUBLISH_VOLUME
time="2020-10-21T09:43:36Z" level=info msg="Enabling controller service capability." capability=LIST_VOLUMES
time="2020-10-21T09:43:36Z" level=info msg="Enabling controller service capability." capability=CREATE_DELETE_SNAPSHOT
time="2020-10-21T09:43:36Z" level=info msg="Enabling controller service capability." capability=LIST_SNAPSHOTS
time="2020-10-21T09:43:36Z" level=info msg="Enabling controller service capability." capability=EXPAND_VOLUME
time="2020-10-21T09:43:36Z" level=info msg="Enabling controller service capability." capability=CLONE_VOLUME
time="2020-10-21T09:43:36Z" level=info msg="Enabling volume access mode." mode=SINGLE_NODE_WRITER
time="2020-10-21T09:43:36Z" level=info msg="Enabling volume access mode." mode=SINGLE_NODE_READER_ONLY
time="2020-10-21T09:43:36Z" level=info msg="Enabling volume access mode." mode=MULTI_NODE_READER_ONLY
time="2020-10-21T09:43:36Z" level=info msg="Enabling volume access mode." mode=MULTI_NODE_SINGLE_WRITER
time="2020-10-21T09:43:36Z" level=info msg="Enabling volume access mode." mode=MULTI_NODE_MULTI_WRITER
time="2020-10-21T09:43:36Z" level=info msg="Added frontend." name=csi
time="2020-10-21T09:43:36Z" level=info msg="Initializing Trident CRD controller frontend." namespace=trident-test
time="2020-10-21T09:43:36Z" level=info msg="Creating event broadcaster."
time="2020-10-21T09:43:36Z" level=info msg="Setting up CRD event handlers."
time="2020-10-21T09:43:36Z" level=info msg="Added frontend." name=crd
time="2020-10-21T09:43:36Z" level=info msg="Activating HTTPS REST frontend." address=":8443"
time="2020-10-21T09:43:36Z" level=info msg="Activating metrics frontend." address=":8001"
time="2020-10-21T09:43:36Z" level=info msg="Activating HTTP REST frontend." address="127.0.0.1:8000"
time="2020-10-21T09:43:36Z" level=warning msg="Persistent state version not found, creating." OrchestratorAPIVersion=1 PersistentStoreVersion=crdv1
time="2020-10-21T09:43:36Z" level=info msg="Added 0 existing volume(s)"
time="2020-10-21T09:43:36Z" level=info msg="Trident bootstrapped successfully."
time="2020-10-21T09:43:36Z" level=info msg="Activating K8S helper frontend."
time="2020-10-21T09:43:36Z" level=info msg="Activating CRD frontend."
time="2020-10-21T09:43:36Z" level=info msg="Activating CSI frontend."
time="2020-10-21T09:43:36Z" level=info msg="Starting Trident CRD controller."
time="2020-10-21T09:43:36Z" level=info msg="Waiting for informer caches to sync."
time="2020-10-21T09:43:36Z" level=info msg="Listening for GRPC connections." name=/plugin/csi.sock net=unix
time="2020-10-21T09:43:36Z" level=info msg="Starting workers."
time="2020-10-21T09:43:36Z" level=info msg="Started workers."
time="2020-10-21T09:43:38Z" level=info msg="Added a new node." handler=AddOrUpdateNode node=worker2.mycluster.example.com
time="2020-10-21T09:43:38Z" level=info msg="Added a new node." handler=AddOrUpdateNode node=worker3.mycluster.example.com
time="2020-10-21T09:43:38Z" level=info msg="Added a new node." handler=AddOrUpdateNode node=master2.mycluster.example.com
time="2020-10-21T09:43:38Z" level=info msg="Added a new node." handler=AddOrUpdateNode node=worker1.mycluster.example.com
time="2020-10-21T09:43:38Z" level=info msg="Added a new node." handler=AddOrUpdateNode node=master3.mycluster.example.com
time="2020-10-21T09:43:39Z" level=info msg="Added a new node." handler=AddOrUpdateNode node=master1.mycluster.example.com
time="2020-10-21T09:45:54Z" level=warning msg="Could not determine controller serial numbers. API status: failed, Reason: Unable to find API: system-node-get-iter, Code: 13005"
time="2020-10-21T09:45:54Z" level=info msg="Storage driver initialized." driver=ontap-nas
time="2020-10-21T09:45:54Z" level=info msg="Created new storage backend." backend="&{initialized:true Config:CommonStorageDriverConfig:&storagedrivers.CommonStorageDriverConfig{Version:1, StorageDriverName:\"ontap-nas\", BackendName:\"test-backend\", Debug:false, DebugTraceFlags:map[string]bool(nil), DisableDelete:false, StoragePrefixRaw:json.RawMessage{0x22, 0x66, 0x62, 0x63, 0x6c, 0x75, 0x73, 0x74, 0x65, 0x72, 0x5f, 0x22}, StoragePrefix:(*string)(0xc000c8e500), SerialNumbers:[]string(nil), DriverContext:\"csi\", LimitVolumeSize:\"100Gi\"} ManagementLIF:\"10.85.86.28\" DataLIF:\"10.85.86.29\" IgroupName:\"\" SVM:\"svm-dev-2\" Username:<REDACTED> Password:<REDACTED> Aggregate:\"clu_ontap_dev_01_VM_DISK_2\" UsageHeartbeat:\"\" QtreePruneFlexvolsPeriod:\"\" QtreeQuotaResizePeriod:\"\" EmptyFlexvolDeferredDeletePeriod:\"\" NfsMountOptions:\"nfsvers=3\" LimitAggregateUsage:\"\" AutoExportPolicy:false AutoExportCIDRs:[]string{\"0.0.0.0/0\", \"::/0\"} OntapStorageDriverPool:storagedrivers.OntapStorageDriverPool{Labels:map[string]string(nil), Region:\"\", Zone:\"\", OntapStorageDriverConfigDefaults:storagedrivers.OntapStorageDriverConfigDefaults{SpaceAllocation:\"true\", SpaceReserve:\"none\", SnapshotPolicy:\"none\", SnapshotReserve:\"\", SnapshotDir:\"false\", UnixPermissions:\"---rwxrwxrwx\", ExportPolicy:\"default\", SecurityStyle:\"unix\", SplitOnClone:\"false\", FileSystemType:\"ext4\", Encryption:\"false\", TieringPolicy:\"\", CommonStorageDriverConfigDefaults:storagedrivers.CommonStorageDriverConfigDefaults{Size:\"1G\"}}} Storage:[]storagedrivers.OntapStorageDriverPool(nil) UseCHAP:false ChapUsername:<REDACTED> ChapInitiatorSecret:<REDACTED> ChapTargetUsername:<REDACTED> ChapTargetInitiatorSecret:<REDACTED>  API:<REDACTED> Telemetry:Telemetry:{  } Plugin:ontap-nas SVM:svm-dev-2 StoragePrefix:fbcluster_ Telemetry.Driver.Name:ontap-nasdone:0xc000d8d380 ticker:&{0xc000787920 {824634126336 643369713034036 86400000000000 0x4c1200 0xc000787920 0 0 1}} stopped:false  physicalPools:map[string]*storage.Pool{\"clu_ontap_dev_01_VM_DISK_2\":(*storage.Pool)(0xc000782240)} virtualPools:map[string]*storage.Pool{}  test-backend  true online map[clu_ontap_dev_01_VM_DISK_2:0xc000782240] map[]}"
time="2020-10-21T09:45:54Z" level=info msg="Newly added backend satisfies no storage classes." backend=test-backend
time="2020-10-21T09:45:54Z" level=info msg="Added a new backend." backend=test-backend handler=AddBackend
time="2020-10-21T09:46:25Z" level=info msg="Replaced storage class so it works with CSI Trident." name=basic newProvisioner=csi.trident.netapp.io oldProvisioner=netapp.io/trident
time="2020-10-21T09:46:25Z" level=info msg="Storage class satisfied by 1 storage pools." storageClass=basic
time="2020-10-21T09:46:25Z" level=info msg="K8S helper added a storage class." name=basic parameters="map[backendType:ontap-nas]" provisioner=csi.trident.netapp.io
time="2020-10-21T09:46:34Z" level=info msg="Found PVC for requested volume pvc-82dea51b-723b-49d6-b4e6-4331e7f76052." UID=82dea51b-723b-49d6-b4e6-4331e7f76052 name=basic namespace=default size="{{1073741824 0} {<nil>} 1Gi BinarySI}" storageClass=basic
time="2020-10-21T09:46:34Z" level=info msg="Found storage class for requested volume pvc-82dea51b-723b-49d6-b4e6-4331e7f76052." name=basic


 oc logs  trident-csi-5k4hz -n trident-test -c driver-registrar
I1021 09:43:36.681588 3319106 main.go:110] Version: v1.3.0-0-g6e9fff3e
I1021 09:43:36.681688 3319106 main.go:120] Attempting to open a gRPC connection with: "/plugin/csi.sock"
I1021 09:43:36.681721 3319106 connection.go:151] Connecting to unix:///plugin/csi.sock
I1021 09:43:36.682288 3319106 main.go:127] Calling CSI driver to discover driver name
I1021 09:43:36.683988 3319106 main.go:137] CSI driver name: "csi.trident.netapp.io"
I1021 09:43:36.684100 3319106 node_register.go:51] Starting Registration Server at: /registration/csi.trident.netapp.io-reg.sock
I1021 09:43:36.684268 3319106 node_register.go:60] Registration Server started at: /registration/csi.trident.netapp.io-reg.sock
I1021 09:43:36.842815 3319106 main.go:77] Received GetInfo call: &InfoRequest{}
I1021 09:43:37.842317 3319106 main.go:77] Received GetInfo call: &InfoRequest{}
I1021 09:43:37.870455 3319106 main.go:87] Received NotifyRegistrationStatus call: &RegistrationStatus{PluginRegistered:true,Error:,}

 oc logs  trident-csi-5k4hz -n trident-test -c trident-main
time="2020-10-21T09:43:36Z" level=info msg="Running Trident storage orchestrator." binary=/usr/local/bin/trident_orchestrator build_time="Tue Sep 22 20:20:23 UTC 2020" version=20.07.1
time="2020-10-21T09:43:36Z" level=info msg="Initializing plain CSI helper frontend."
time="2020-10-21T09:43:36Z" level=info msg="Added frontend." name=plain_csi_helper
time="2020-10-21T09:43:36Z" level=info msg="Initializing CSI frontend." name=worker3.mycluster.example.com version=20.07.1
time="2020-10-21T09:43:36Z" level=info msg="Enabling node service capability." capability=STAGE_UNSTAGE_VOLUME
time="2020-10-21T09:43:36Z" level=info msg="Enabling node service capability." capability=EXPAND_VOLUME
time="2020-10-21T09:43:36Z" level=info msg="Enabling node service capability." capability=GET_VOLUME_STATS
time="2020-10-21T09:43:36Z" level=info msg="Enabling volume access mode." mode=SINGLE_NODE_WRITER
time="2020-10-21T09:43:36Z" level=info msg="Enabling volume access mode." mode=SINGLE_NODE_READER_ONLY
time="2020-10-21T09:43:36Z" level=info msg="Enabling volume access mode." mode=MULTI_NODE_READER_ONLY
time="2020-10-21T09:43:36Z" level=info msg="Enabling volume access mode." mode=MULTI_NODE_SINGLE_WRITER
time="2020-10-21T09:43:36Z" level=info msg="Enabling volume access mode." mode=MULTI_NODE_MULTI_WRITER
time="2020-10-21T09:43:36Z" level=info msg="Added frontend." name=csi
time="2020-10-21T09:43:36Z" level=info msg="Added 0 existing volume(s)"
time="2020-10-21T09:43:36Z" level=warning msg="Couldn't retrieve volume transaction logs: Unable to find key"
time="2020-10-21T09:43:36Z" level=info msg="Trident bootstrapped successfully."
time="2020-10-21T09:43:36Z" level=info msg="Activating plain CSI helper frontend."
time="2020-10-21T09:43:36Z" level=info msg="Activating CSI frontend."
time="2020-10-21T09:43:36Z" level=error msg="could not read transactions" error="Unable to find key"
time="2020-10-21T09:43:36Z" level=info msg="Listening for GRPC connections." name=/plugin/csi.sock net=unix
time="2020-10-21T09:43:36Z" level=info msg="Discovered IP addresses." IP Addresses="[10.131.0.1 192.168.10.32]"
time="2020-10-21T09:55:23Z" level=error msg="GRPC error: rpc error: code = Internal desc = error mounting NFS volume 10.85.86.29:/fbcluster_pvc_82dea51b_723b_49d6_b4e6_4331e7f76052 on mountpoint /var/lib/kubelet/pods/93e13576-5ba0-4283-af61-eb9af5f7682b/volumes/kubernetes.io~csi/pvc-82dea51b-723b-49d6-b4e6-4331e7f76052/mount: exit status 32"
time="2020-10-21T09:55:24Z" level=error msg="GRPC error: rpc error: code = Internal desc = error mounting NFS volume 10.85.86.29:/fbcluster_pvc_82dea51b_723b_49d6_b4e6_4331e7f76052 on mountpoint /var/lib/kubelet/pods/93e13576-5ba0-4283-af61-eb9af5f7682b/volumes/kubernetes.io~csi/pvc-82dea51b-723b-49d6-b4e6-4331e7f76052/mount: exit status 32"
time="2020-10-21T09:55:25Z" level=error msg="GRPC error: rpc error: code = Internal desc = error mounting NFS volume 10.85.86.29:/fbcluster_pvc_82dea51b_723b_49d6_b4e6_4331e7f76052 on mountpoint /var/lib/kubelet/pods/93e13576-5ba0-4283-af61-eb9af5f7682b/volumes/kubernetes.io~csi/pvc-82dea51b-723b-49d6-b4e6-4331e7f76052/mount: exit status 32"
time="2020-10-21T09:55:27Z" level=error msg="GRPC error: rpc error: code = Internal desc = error mounting NFS volume 10.85.86.29:/fbcluster_pvc_82dea51b_723b_49d6_b4e6_4331e7f76052 on mountpoint /var/lib/kubelet/pods/93e13576-5ba0-4283-af61-eb9af5f7682b/volumes/kubernetes.io~csi/pvc-82dea51b-723b-49d6-b4e6-4331e7f76052/mount: exit status 32"
time="2020-10-21T09:55:31Z" level=error msg="GRPC error: rpc error: code = Internal desc = error mounting NFS volume 10.85.86.29:/fbcluster_pvc_82dea51b_723b_49d6_b4e6_4331e7f76052 on mountpoint /var/lib/kubelet/pods/93e13576-5ba0-4283-af61-eb9af5f7682b/volumes/kubernetes.io~csi/pvc-82dea51b-723b-49d6-b4e6-4331e7f76052/mount: exit status 32"
time="2020-10-21T09:55:39Z" level=error msg="GRPC error: rpc error: code = Internal desc = error mounting NFS volume 10.85.86.29:/fbcluster_pvc_82dea51b_723b_49d6_b4e6_4331e7f76052 on mountpoint /var/lib/kubelet/pods/93e13576-5ba0-4283-af61-eb9af5f7682b/volumes/kubernetes.io~csi/pvc-82dea51b-723b-49d6-b4e6-4331e7f76052/mount: exit status 32"
time="2020-10-21T09:55:55Z" level=error msg="GRPC error: rpc error: code = Internal desc = error mounting NFS volume 10.85.86.29:/fbcluster_pvc_82dea51b_723b_49d6_b4e6_4331e7f76052 on mountpoint /var/lib/kubelet/pods/93e13576-5ba0-4283-af61-eb9af5f7682b/volumes/kubernetes.io~csi/pvc-82dea51b-723b-49d6-b4e6-4331e7f76052/mount: exit status 32"
time="2020-10-21T09:56:27Z" level=error msg="GRPC error: rpc error: code = Internal desc = error mounting NFS volume 10.85.86.29:/fbcluster_pvc_82dea51b_723b_49d6_b4e6_4331e7f76052 on mountpoint /var/lib/kubelet/pods/93e13576-5ba0-4283-af61-eb9af5f7682b/volumes/kubernetes.io~csi/pvc-82dea51b-723b-49d6-b4e6-4331e7f76052/mount: exit status 32"
time="2020-10-21T09:57:32Z" level=error msg="GRPC error: rpc error: code = Internal desc = error mounting NFS volume 10.85.86.29:/fbcluster_pvc_82dea51b_723b_49d6_b4e6_4331e7f76052 on mountpoint /var/lib/kubelet/pods/93e13576-5ba0-4283-af61-eb9af5f7682b/volumes/kubernetes.io~csi/pvc-82dea51b-723b-49d6-b4e6-4331e7f76052/mount: exit status 32"
```
Looking at service on worker where the pod is deployed
```
sudo systemctl list-units | grep rpc
rpc-statd.service                                                                                                                                         loaded active running   NFS status monitor for NFSv2/3 locking.                                                                                  
rpcbind.service                                                                                                                                           loaded active running   RPC Bind                                                                                                                 
rpcbind.socket                                                                                                                                            loaded active running   RPCbind Server Activation Socket                                                                                         
rpcbind.target                                                                                                                                            loaded active active    RPC Port Mapper    
```

<img width="1415" alt="Screenshot 2020-10-21 at 18 52 40" src="https://user-images.githubusercontent.com/10832015/96754160-313b6200-13d1-11eb-9476-9142d748ba99.png">

**Comment by gnarl** [comment]
Hi @fckbo,

Did you follow the[ instructions](https://netapp-trident.readthedocs.io/en/stable-v20.07/kubernetes/operations/tasks/worker.html#nfs) for worker node prep?

If you have already done the worker node prep then try running the mount command from the worker node. If this isn't successful then there is a configuration issue outside of Trident that needs to be resolved.

**Comment by balaramesh** [comment]
@fckbo the export policy on your SVM may not permit NFS shares to mount on the nodes in your cluster. You can test this out with a manual mount from a worker node. You can manage export policies outside of Trident or use [dynamic export policies](https://netapp-trident.readthedocs.io/en/stable-v20.07/kubernetes/operations/tasks/backends/ontap/ontap-nas/dynamic-export-policies.html).

**Comment by fckbo** [comment]
@gnarl,

in fact as mentioned above the worker nodes are RHCOS ones and  the instructions are mentioning the following : 
"Recent versions of RedHat CoreOS have both installed by default. You must ensure that the NFS and iSCSI services are started up during boot time."

When logging on the worker on which the test pod is deployed, I can do a show mount -e <ip @ of the ONTAP> that displays the list of pv:
```
showmount -e 10.85.86.29
Export list for 10.85.86.29:
/fbcluster_pvc_82dea51b_723b_49d6_b4e6_4331e7f76052 (everyone)
/                                                   (everyone)
```

**Comment by fckbo** [comment]
@balaramesh,

thx for your hint, I have put a rule to allow any access from 0.0.0.0/0 on the SVM as I thought access control might be the issue... ref the diagram of the setup above, I did a test on the underlying CentOS8 machine to check that I could mount the pv created by trident from that machine
```
mkdir -p /test
mount -t nfs 10.85.86.29:/fbcluster_pvc_82dea51b_723b_49d6_b4e6_4331e7f76052 /test
ls -la /test
total 8
drwxrwxrwx.  2 root root 4096 Oct 21 11:47 .
drwxr-xr-x. 22 root root 4096 Oct 21 19:07 ..
```

**Comment by balaramesh** [comment]
@fckbo do you still observe this issue with a newly created PVC/pod?

**Comment by fckbo** [comment]
yes, issue still there :-(

**Comment by wonderland** [comment]
Did you try the NFS mount on the worker node as well? I only see the showmount from  the worker node and the successful mount from the Centos machine. Anything on the network side that could play a role? Firewalls, NAT, ...?

**Comment by fckbo** [comment]
@wonderland,

NAT was the issue, I had only configured NFSv3 on the ONTAP and also had indicated     "nfsMountOptions": "nfsvers=3" in the backend.json provided above. It turned out that I did not find how to disable '-mount-rootonly' using the ONTAP System Management UI https://docs.netapp.com/ontap-9/index.jsp?topic=%2Fcom.netapp.doc.cdot-famg-nfs%2FGUID-FF0434AB-B2F4-4EF0-AA76-52EBBC40586E.html (I don't have the CLI ...). I end-up adding NFS V4 & V4.1 and did update the backend to remove the  "nfsMountOptions": "nfsvers=3" ... and it worked.

@wonderland, @balamaresh, @gnarl,
a big thank to you guys for your help !
